scripts/mdbook-forc-documenter/src/plugins.rs

```diff
@@ -0,0 +1,28 @@
+use std::fs;
+use std::path::PathBuf;
+
+fn get_sway_path() -> PathBuf {
+    let mut curr_path = std::env::current_dir().unwrap();
+    loop {
+        if curr_path.ends_with("sway") {
```

Comment:
Just because the repo's name is `sway` does not mean that a local clone of the repo is. E.g. while less common, there are many git users out there that do stuff like `git clone https://github.com/fuellabs/sway Sway` or give it some other name that suits them.

Instead, can we make this a `find_forc_plugins_dir` where, instead of looking for a path ending with `sway`, we keep looping through parents (as we currently do) until the directory at `curr_path` contains a `forc-plugins` subdirectory and return that?